test/support/test_source.ex

```diff
@@ -6,7 +6,6 @@ defmodule Membrane.H264.Support.TestSource do
   def_options mode: []
 
   def_output_pad :output,
-    demand_mode: :auto,
     mode: :push,
```

Comment:
Why?

Reply:
it does not make any sense, to have `demand_mode: :auto` with `mode: :push`. Beyond this, such a pad definition raises in core 0.12.1

Reply:
And `:auto` is not allowad in sources output pads